assets/js/map.js

```diff
@@ -0,0 +1,203 @@
+(function(exports) {
+
+  var map = d3.select("#team-map")
+        .classed("loading", true),
+      status = map.append("p")
+        .html("Loading the map...");
+
+  // names and geographic locations of airport codes
+  var locations = [
```

Comment:
Soon we'll add a `_data/locations.yml` file that'll have the coordinates. We can then make this file a Jekyll-processed template that spits the location codes and coordinates into this array, and/or just pull them from the Hub's `/api/locations/api.json` endpoint.